src/persist-client/src/batch.rs

```diff
@@ -1098,6 +1098,11 @@ impl<T: Timestamp + Codec64> BatchParts<T> {
             let blob = Arc::clone(&blob);
             let metrics = Arc::clone(&metrics);
             let writer_key = cfg.writer_key.clone();
+            // Don't spill "unordered" runs to S3, since we'll split them up into many length-1
```

Comment:
Initially when I read this I thought "length-1" meant "length minus 1", but now I realize it means "length of 1"?

Reply:
Correct - updated some comments!